src/charts/scatter-plot.js

```diff
@@ -1203,6 +1213,23 @@ define(function(require) {
             return this;
         };
 
+        /**
+         * exposes ability to set the formatter of x-axis values
+         * @param  {string} _x          type of x-axis formatter
+         * @value 'number'
+         * @value 'time'
+         * @return {string | module}    current xAxisFormatType or Chart module to chain calls
+         * @public
+         */
+        exports.xAxisFormatType = function(_x) {
```

Comment:
We'll need a simple API accessor test for this one

Reply:
Yes please, this can be added anywhere within this `describe` block, look at the examples for code referral: https://github.com/britecharts/britecharts/blob/master/test/specs/scatter-plot.spec.js#L525

Reply:
thank you, for pointing out the relevant test section